platform/iphone/gl_view.mm

```diff
@@ -284,17 +284,27 @@ - (id)initGLES {
 			kEAGLColorFormatRGBA8,
 			kEAGLDrawablePropertyColorFormat,
 			nil];
-
-	// Create our EAGLContext, and if successful make it current and create our framebuffer.
-	context = [[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES3];
-
-	if (!context || ![EAGLContext setCurrentContext:context] || ![self createFramebuffer]) {
+	
+	// Create a context based on the gl driver from project settings
+	if (GLOBAL_GET("rendering/quality/driver/driver_name") == "GLES3") {
+		context = [[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES3];
+		NSLog(@"Setting up an OpenGL ES 3 context. Based on Project Settings \"rendering/quality/driver/driver_name\"");
```

Comment:
Might need to add something here to handle the fallback Project Setting

Reply:
Looks like we’re clear here. I double checked the initialization and it handles fallback as part of driver init.